### ISSUE TITLE
Fail to detect error for @CheckReturnValue for org.assertj.core.api.Assertions.assertThat()

### ISSUE THREAD

**Issue by bloussou** [spotbugs/spotbugs-maven-plugin#301]
Hello,
I tried to set up `spotbugs-maven-plugin` to detect misuse of `Assertions.assertThat()` of assertJ as explained in [assertJ doc](https://assertj.github.io/doc/#assertj-core-incorrect-usage) but no errors is detected for this...

I'm using assertJ core `3.18.1`.
Here are my settings : 
```xml
<plugin>
  <groupId>com.github.spotbugs</groupId>
  <artifactId>spotbugs-maven-plugin</artifactId>
  <version>4.2.0</version>
  <configuration>
    <effort>Max</effort>
    <includeFilterFile>spotbugsIncludeFilter.xml</includeFilterFile>
    <includeTests>true</includeTests>
  </configuration>
  <executions>
    <execution>
      <goals>
        <goal>check</goal>
      </goals>
    </execution>
  </executions>
</plugin>
```
spotbugsIncludeFilter.xml :
```xml
<?xml version="1.0" encoding="UTF-8"?>
<FindBugsFilter
		xmlns="https://github.com/spotbugs/filter/3.0.0"
		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
		xsi:schemaLocation="https://github.com/spotbugs/filter/3.0.0 https://raw.githubusercontent.com/spotbugs/spotbugs/3.1.0/spotbugs/etc/findbugsfilter.xsd">
	<!-- All bugs in test classes, except for JUnit-specific bugs -->
	<Match>
		<Class name="~.*\.Test.*" />
	</Match>
</FindBugsFilter>
```
Thanks for your help

**Comment by hazendaz** [comment]
Likely related to this https://github.com/spotbugs/spotbugs/issues/429.

**Comment by hazendaz** [comment]
closing as not an issue with the maven plugin but rather spotbugs and open ticket there.
